# 1st run
walker@MSI MINGW64 /r/Projects/devops/udacity-project4 (dev)
$ ./run_kubernetes.sh
pod/devops-ml created
NAME        READY   STATUS    RESTARTS   AGE
devops-ml   0/1     Pending   0          1s
error: unable to forward port because pod is not running. Current status=Pending

# 2nd run after the pod status is Running
walker@MSI MINGW64 /r/Projects/devops/udacity-project4 (dev)
$ ./run_kubernetes.sh
NAME        READY   STATUS    RESTARTS   AGE
devops-ml   1/1     Running   0          10m
Forwarding from 127.0.0.1:8080 -> 80
Forwarding from [::1]:8080 -> 80
Handling connection for 8080

# make_prediction.sh run
walker@MSI MINGW64 /r/Projects/devops/udacity-project4 (dev)
$ kubectl logs devops-ml
 * Serving Flask app "app" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 154-462-921
[2023-08-24 23:01:14,389] INFO in app: JSON payload:
{'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2023-08-24 23:01:14,980] INFO in app: Inference payload DataFrame:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2023-08-24 23:01:14,996] INFO in app: Scaling Payload:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
127.0.0.1 - - [24/Aug/2023 23:01:15] "POST /predict HTTP/1.1" 200 -
